src/ui/hexedit.cc

```diff
@@ -643,10 +660,53 @@ QString HexEdit::addressAsText(qint64 pos) {
 }
 
 QString HexEdit::asciiRepresentationFromByte(uint64_t byte_val) {
-  if (byte_val >= 0x20 && byte_val < 0x7f) {
-    return QChar::fromLatin1(byte_val);
+  if (byte_val > 0xff) {
+    return ".";
+  }
+  if (windows1250_codec_ != nullptr &&
+      unprintables_mode_ == UnprintablesMode::Windows_1250) {
+    char a = veles::util::misc::ucharToChar(byte_val);
+    QChar unicode_repr = windows1250_codec_->toUnicode(&a, 1).at(0);
+
+    bool is_undefined_windows1250 = byte_val == 0x81 || byte_val == 0x83 ||
+                                    byte_val == 0x88 || byte_val == 0x90 ||
+                                    byte_val == 0x98;
+
+    bool is_whitespace = unicode_repr.isSpace() || unicode_repr.isNull();
+
+    if (is_whitespace || is_undefined_windows1250) {
```

Comment:
Please add a check for 0x7f here, it decodes to `DEL` and it's not printable.

Reply:
Done.